Podfile

```diff
@@ -16,6 +16,7 @@ def ios_pods
     pod 'PinpointKit', :git => 'https://github.com/Lickability/PinpointKit', :commit => 'dd5731c'
     pod 'DZNEmptyDataSet', '1.8.1'
     pod 'Hero'
+    pod 'SimpleRoundedButton'
```

Comment:
Do we really need a pod for that? You can also call `layer.cornerRadius` which is basically everything this pod does.